peps/pep-0776.rst

```diff
@@ -2,13 +2,12 @@ PEP: 776
 Title: Emscripten Support
 Author: Hood Chatham <roberthoodchatham at gmail.com>
 Sponsor: Łukasz Langa <lukasz at python.org>
-Discussions-To: https://discuss.python.org/t/84996
+Discussions-To: https://discuss.python.org/t/86276
 Status: Draft
 Type: Standards Track
```

Comment:
My understanding is that the move to tier three has already been approved, and the PEP notes that it doesn't seek to specify anything, so perhaps informational is now more appropriate?
```suggestion
Type: Informational
```

Reply:
Well it is similar to https://peps.python.org/pep-0738/ and https://peps.python.org/pep-0730/ which are standards track peps. But I am open to making it informational if there is a consensus that that is the right way to go, since we are in a slightly different situation than android and ios were.